### PR TITLE
New SIGNL4 alerting article

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a new SIGNL4 alerting article originally submitted by @rons4 in PR #3142. It adds this article:
https://help.sumologic.com/docs/alerts/webhook-connections/signl4

Issue number: #3111 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/alerts/webhook-connections/signl4.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+sidebar_label: SIGNL4
+description: Send alerts to SIGNL4 from scheduled searches.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('https://raw.githubusercontent.com/signl4/signl4-logo/main/signl4.png')} alt="Thumbnail icon" width="50"/>
+
+
+[Webhook connections](/docs/alerts/webhook-connections/set-up-webhook-connections) rely on HTTP endpoints that tell Sumo Logic where to send data. You can set up any number of connections.
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/signl4.md:13]:
```suggestion
You can set up [webhook connections](/docs/alerts/webhook-connections/set-up-webhook-connections) rely on HTTP endpoints that tell Sumo Logic where to send data.
```

File: docs/alerts/webhook-connections/signl4.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+## Sumo Logic to SIGNL4 Integration
+
+Sumo Logic alerts can send webhook alerts to SIGNL4, a mobile alerting and incident management solution that determines the right people to alert based on your on-call schedule, severity, and topic.
+
+To add a Sumo Logic integration in SIGNL4, do the following:
+
+1. Go to the [SIGNL4 Integration Hub](https://account.signl4.com/manage/IntegrationHub?tabName=installedApps).
+1. Select an existing webhook integration or create a new one for the Sumo Logic integration.
+1. In the integration tile copy the URL including integration or team secret.
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/signl4.md:23]:
```suggestion
1. In the integration tile, copy the URL including integration or team secret.
```

File: docs/alerts/webhook-connections/signl4.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+You need the webhook URL for the configuration in Sumo Logic.
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/signl4.md:25]:
```suggestion
You'll need the webhook URL for Sumo Logic configuration, which you'll do in the next section.
```

File: docs/alerts/webhook-connections/signl4.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+This section demonstrates how to create a webhook connection from Sumo Logic to SIGNL4.
+
+To create a webhook:
+
+1. In Sumo Logic, go to **Manage Data > Alerts > Connections**.
+1. Click **+ Add** and choose **Webhook** as connection type.
+1. For the name, enter **Sumo Logic SIGNL4** and give an optional description to the connection.
+1. Paste the SIGNL4 webhook URL (from the step above) into the **URL** field.
+1. Enter the following content in the **Alert Payload** field:
+    ```json
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/signl4.md:51]:
```suggestion
    ```
```

File: docs/alerts/webhook-connections/signl4.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+This section demonstrates how to create a webhook connection from Sumo Logic to SIGNL4.
+
+To create a webhook:
+
+1. In Sumo Logic, go to **Manage Data > Alerts > Connections**.
+1. Click **+ Add** and choose **Webhook** as connection type.
+1. For the name, enter **Sumo Logic SIGNL4** and give an optional description to the connection.
+1. Paste the SIGNL4 webhook URL (from the step above) into the **URL** field.
+1. Enter the following content in the **Alert Payload** field:
+    ```
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/signl4.md:51]:
```suggestion
    ```json
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
A few comments

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce. I committed your suggestions.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**